.github/workflows/run_torchbench.yml

```diff
@@ -36,7 +36,9 @@ jobs:
           # shellcheck disable=SC1091
           . "${HOME}"/anaconda3/etc/profile.d/conda.sh
           conda activate pr-ci
-          conda install -y numpy requests ninja pyyaml mkl mkl-include setuptools cmake cffi typing_extensions \
+          # pin cmake version to 3.22 since 3.23 breaks pytorch build
+          # see details at: https://github.com/pytorch/pytorch/issues/74985
+          conda install -y numpy requests ninja pyyaml mkl mkl-include setuptools cmake=3.22 cffi typing_extensions \
```

Comment:
Maybe a good follow up to this would be to just pin all of these particular dependencies

Reply:
One problem of doing that is we need to keep consistent with pytorch/benchmark workflows such as in https://github.com/pytorch/benchmark/blob/main/.github/workflows/v2-bisection.yml#L31. Perhaps it is better to create a script in pytorch/benchmark to manage these dependencies in a unified manner.